### PR TITLE
Clean up JS code a bit

### PR DESCRIPTION
**Pull request by eddiemonge**
Optimized the jQuery a bit. Also removed the delete column as it didnt let you do anything anyway

### CODE REVIEW

**Comment by bunchjesse** [comment]
Thanks Eddie. Give me some time to test your diff and I'll merge if I find no issues.

File: ext.disable_template_editor.php [hunk 1]
```diff
@@ -27,11 +27,11 @@
 class Disable_template_editor_ext
 {
 	public $settings 		= array();
-	public $description		= 'Disables the EE template editor for users to control templates via source control. This prevents folks from creating out of sync issues between the server and repo.';
+    public $description		= 'Disables the EE template editor for users to control templates via source control. This prevents folks from creating out of sync issues between the server and repo.';
```
Comment (bunchjesse) [inline ext.disable_template_editor.php]: Please fix this indentation.

File: ext.disable_template_editor.php [hunk 2]
```diff
@@ -86,19 +86,27 @@ public function activate_extension()
...
+            '    ' .
+            '    $(".templateEditorTop").append("<div style=\'clear: left\'>All templates and template groups must be created and edited via the templates folder on the filesystem.</div>").children("h2").html("Template Management (Under Source Control)");' .
+            '    $tmplTable.find("input[name=template_name]").prop("disabled", "disabled");' .
+            '    $tmplTable.find(".template_manager_template_name").html(function(i,old) {' .
+            '        return old + " (Read Only)";' .
+            '    });' .
+            '    $tmplTable.find("td.cellRight, th.cellRight").remove();' .
+            '});';
 
 		return !$this->EE->extensions->last_call ? $str : $this->EE->extensions->last_call . $str;
```
Comment (bunchjesse) [inline ext.disable_template_editor.php:111]: Could you align this indention with the `$str` declaration?

**Comment by eddiemonge** [comment]
@bunchjesse Updated the formatting

**Comment by bunchjesse** [comment]
@eddiemonge Looks good. I'll test this tonight (hopefully) and merge it if I find no issues. Thanks again for submitting the pull request.

**Comment by eddiemonge** [comment]
merged?

**Comment by eddiemonge** [comment]
giving up on getting this landed

**Pull request closed without merging**